From 2a7510b8ce1a6ac727b63c2e68ba9c8d1ff88f93 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Wed, 24 Jan 2024 19:03:45 +0200
Subject: [PATCH] [SW-170781] gaudi2: sec: use new enum name

This patch uses new enum name HL_SEC_LBW_NIC_CFG
for HL_SEC_LBW_RSVD.

HL_SEC_LBW_RSVD will be removed from spec header.

Change-Id: Iaccb599b30c9fd447edf8777116ec6eb24120c9e
Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 2ccf56a..174c9ef 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -173,7 +173,7 @@
 	HL_SEC_LBW_DBI_1,
 	HL_SEC_LBW_DBI_2,
 	HL_SEC_LBW_PSOC_ETR_RWD,
-	HL_SEC_LBW_RSVD
+	HL_SEC_LBW_NIC_CFG
 };
 
 #define SRAM_RSVD_BASE		0x1000008000000000ull
